### DISCUSSION TITLE
Welcome to laravel-permission-mongodb Discussions!

### DISCUSSION THREAD

**Discussion by mostafamaklad** [mostafamaklad/laravel-permission-mongodb#121] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zamin-pc** [comment]
Hi,

I am looking to upgrade my existing Laravel 9 project to Laravel 11. Could you please let me know when an update for Laravel 11 compatibility is expected to be released? Any estimated date would be greatly appreciated.

Thanks.

**Comment by zamin-pc** [comment]
Hi @mostafamaklad ,

I’m currently awaiting an update for compatibility with Laravel 10 or 11. Could you please let me know when the package will be updated to support these versions?

Thank you in advance!
